File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/TestPos.java
package org.firstinspires.ftc.teamcode.opmodes;

import com.qualcomm.robotcore.eventloop.opmode.Autonomous;

import static org.firstinspires.ftc.teamcode.opmodes.MovementMacros.ShootPOWERSHOT;
import static org.firstinspires.ftc.teamcode.robot.WoENrobot.openCVNode;
import static org.firstinspires.ftc.teamcode.robot.WoENrobot.shooter;
import static org.firstinspires.ftc.teamcode.robot.WoENrobot.spinOnce;

@Autonomous(name = "Pos test")
public class TestPos extends AutonomousOpMode {
    @Override
    public void main() {
        ShootPOWERSHOT();
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/TestPos.kt
package org.firstinspires.ftc.teamcode.opmodes

import com.qualcomm.robotcore.eventloop.opmode.Autonomous
import org.firstinspires.ftc.teamcode.opmodes.MovementMacros.ShootPOWERSHOTAngle

@Autonomous(name = "Pos test")
class TestPos : AutoOpMode() {
    override fun main() {
        ShootPOWERSHOTAngle()
    }
}